FabLabKasse/kassenbuch.py

```diff
@@ -859,8 +870,111 @@ def writerows(self, rows):
         print("Gespeichert. Kundennummer lautet: " + str(kunde.id))
 
     elif arguments['client'] and arguments['edit']:
-        print("Work-in-progress...")
-        print("IMPLEMENT ME!")
+        try:
+            kunde = Kunde.load_from_name(arguments['<name>'], k.cur)
+        except NoDataFound:
+            print(u"Konnte keinen Kunde unter '%s' finden." % arguments['<name>'])
+            sys.exit(2)
+
+        # Name
+        while True:
+            name = raw_input('Name (ohne Leer- und Sonderzeichen!) [%s]: ' % kunde.name)
```

Comment:
Wenn `kunde.name` unicode Zeichen enthält, crashts hier und ebenso in [L956](https://github.com/fau-fablab/FabLabKasse/pull/97/files#diff-5aa974ab3c8fdd7bd18b70d6c09ed9afR956) und in [L964](https://github.com/fau-fablab/FabLabKasse/pull/97/files#diff-5aa974ab3c8fdd7bd18b70d6c09ed9afR964). Man könnte `kunde.name` ersetzen mit `kunde.name.encode(sys.stdout.encoding)` oder man findet eine bessere Lösung.